src/plot_api/plot_api.js

```diff
@@ -3856,9 +3852,25 @@ function makePlotFramework(gd) {
         .style('top', '0px')
         .style('right', '0px');
 
+    fullLayout._lastBBox = gd.getBoundingClientRect();
+
     gd.emit('plotly_framework');
 }
 
+// determines if the graph div requires a recalculation of its inverse matrix transforms by comparing old + new bounding boxes.
+function recalculateTransformInverseIfNecessary(gd, newBBox = null) {
```

Comment:
Declaring an optional argument e.g. `newBBox` is not supported in ES5.
Please run the command below before commits
```npm run test-syntax```
Or alternatively enable `eslint` in your editor (e.g. vs-code) to catch similar errors.
Thanks.